lib/accordion/accordion.test.tsx

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 
 import Accordion from './accordion';
 
+import { NAMESPACE } from '../utilities/ts/constants';
+
 import { mount } from 'enzyme';
 
 let wrapper;
 
-describe('<Acordion />', () => {
+describe(`<Acordion />`, () => {
```

Comment:
Typo <Acordion/> instead of <Accordion />

Reply:
Hey bud, this one isn't right... It is 'accordion'.

Reply:
OH! Now I understand...

Reply:
It should be more descriptive. ahha